### PR TITLE
Attempting PR to main branch

### PR DESCRIPTION
**Pull request by epauluk92**
I'm hoping this is correct; I added two sets of files to my fork. One set is a simple viewshed analysis (AnnieP.py/json) and the other is an LCP analysis I wasn't sure would work, as it's more complicated. Both ran in binder, but not sure how well that translates to the actual table.

### CODE REVIEW

**Comment by wenzeslaus** [comment]
The checks were not triggered for some reason (something internal to GitHub, not related to us or the PR). I will try if closing and opening the PR helps (it should but it did not for me this morning for another PR).

**Comment by petrasovaa** [comment]
Closed and reopened because I want to see the rendering artifact that expired.

File: activities/AnnieP_LCPView.py [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+
+def main():
+    env = os.environ.copy()
+    env["GRASS_OVERWRITE"] = "1"
+    elevation = "elev_lid792_1m"
+    start = [638469, 220070]
+    end = [638928, 220472]
+
+    LCP(elevation, start, end, env)  # run the LCP command
+    run_viewshed(scanned_elev=elevation, env=env)  # run the viewshed from the end
```
Comment (petrasovaa) [inline activities/AnnieP_LCPView.py]:
```suggestion
    start = [638469, 220070]
    end = [638928, 220472]
    elev_resampled = "elev_resampled"
    gs.run_command("g.region", raster=elevation, res=4, flags="a", env=env)
    gs.run_command("r.resamp.stats", input=elevation, output=elev_resampled, env=env)
    
    LCP(elev_resampled, start, end, env)  # run the LCP command
    run_viewshed(scanned_elev=elev_resampled, env=env)  # run the viewshed from the end
```

File: activities/AnnieP_LCPView.json [hunk 2]
```diff
@@ -0,0 +1,21 @@
+{
+  "tasks": [
+    {
+      "layers": [
+        ["d.rast", "map=viewshed"],
+        ["d.rast", "map=cost"]
```
Comment (petrasovaa) [inline activities/AnnieP_LCPView.json]:
```suggestion
        ["d.rast", "map=cost"] ,
        ["d.rast", "map=viewshed", "values=1"]

```

File: activities/AnnieP_LCPView.py [hunk 3]
```diff
@@ -0,0 +1,60 @@
...
+def main():
+    env = os.environ.copy()
+    env["GRASS_OVERWRITE"] = "1"
+    elevation = "elev_lid792_1m"
+    start = [638469, 220070]
+    end = [638928, 220472]
+    elev_resampled = "elev_resampled"
+    gs.run_command("g.region", raster=elevation, res=4, flags="a", env=env)
+    gs.run_command("r.resamp.stats", input=elevation, output=elev_resampled, env=env)
+
```
Comment (petrasovaa) [inline activities/AnnieP_LCPView.py]:
```suggestion

```

**Pull request merged**